test/cppsuite/test_harness/workload/thread_context.h

```diff
@@ -136,15 +136,17 @@ class thread_context {
     /*
      * Generic update function, takes a collection_id and key, will generate the value.
      *
-     * Returns true if a rollback is required.
+     * Returns true if the operation was successful, a return value of false implies the transaction
```

Comment:
Use the imperative tense in description, `Return` *

Reply:
For all of the instances of returns in this file or just these two?

Reply:
Could be great to fix the whole file since you're on it.